src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -18,6 +18,8 @@ public sealed class CSharpParseOptions : ParseOptions, IEquatable<CSharpParseOpt
         /// </summary>
         public static readonly CSharpParseOptions Default = new CSharpParseOptions();
 
+        private IReadOnlyDictionary<string, string> _features;
```

Comment:
It can't be readonly due to WithLanguageVersion assigning it after construction. WithLanguageVersion can't pass it into the constructor, either, because it's a public API and I assume shouldn't be changed. (CompilationOptions does the same thing, with a mutable field hidden under a With* method)

The only place it should be used is in the Features property, which is of type IReadOnlyDictionary, so I figured it should be of that type.